cmd/soroban-rpc/internal/db/ledgerentry_test.go

```diff
@@ -196,6 +437,23 @@ func getContractDataLedgerEntry(data xdr.ContractDataEntry) (xdr.LedgerKey, xdr.
 	return key, entry
 }
 
+func getContractCodeLedgerEntry(code xdr.ContractCodeEntry) (xdr.LedgerKey, xdr.LedgerEntry) {
```

Comment:
given that this is a testing code, it would be cleaner to add the `t *testing.T` to its input parameters and `require.NotNil` the error below instead of panicing.